students/scott/session01/break_me.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+
+# Write a function that generates a NameError
+
+def generate_name_error():
+	print(paul_blart)
+
+generate_name_error()
+
+
+
+
+#Write a function that generates a TypeError
+
+a = 1
+b = "threads"
+
+this_thing = a + b
+
+def generate_type_error():
+	print("this_thing")
+
+generate_type_error()
+
+
+
+
+#Write a function that generates a SyntaxError
+
+def generate_syntax_error():
+#	print)"foo" and "bar")
+
+generate_syntax_error()
+
+
+
+
+#Write a function that generates a AttributeError
+
+int - 4
+int.append(5)
+print(int)
```

Comment:
I assume you meant int = 4, in which case the int.append(5) will give you an AttributeError, since an int does not have the attribute append. If you put the both lines in a function then you would get the error when you run the function. Btw, int is a built-in, so you should not use it for a variable name.